src/views/Vote/CreateProposal.tsx

```diff
@@ -1,7 +1,103 @@
+  const [textarea, setTextArea] = useState({
+    value: '',
+    length: 0,
+  })
+
+  const handleChange = (e) => {
+    setTitle(e.target.value)
+  }
+  const handleChangeDiscussion = (e) => {
+    setDisscussion(e.target.value)
+  }
+  const handleTextArea = (e) => {
+    setTextArea({ ...textarea, value: e.target.value, length: textarea?.value?.length })
+  }
+  const handleDrop = (e) => {
+    e.preventDefault()
+    e.stopPropagation()
+    const imageFile = e.dataTransfer.files
+  }
+  const handleDragOver = (e) => {
+    e.preventDefault()
+  }
+
+  return (
+    <>
+      <Flex sx={styles.main}>
+        <Box>
+          <ArrowBackIcon name="ArrowBack" color="text" width="25px" height="15px" />
+          <a href="https://vote.apeswap.finance/#/about">Back</a>
+          <Box sx={styles.firstSection}>
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Svg icon="info" width="15" />
+              <Text sx={{ fontSize: '14px', marginLeft: '10px' }}>
+                You need to have a minimum of 2.5K GNANA in order to submit a proposal.
+              </Text>
+            </Box>
+            <Link href="https://vote.apeswap.finance/#/about">
+              <Text sx={{ fontSize: '14px', marginTop: '5px' }}>Learn More</Text>
+            </Link>
+          </Box>
+          {preview ? (
+            <>
+              <Text sx={{ marginBottom: '10px', display: 'block' }}>Title</Text>
 
+              <Input onChange={handleChange} value={title} />
+
+              <Flex sx={{ justifyContent: 'space-between', marginTop: '15px' }}>
+                <Text sx={{ marginBottom: '15px' }}>Description (optional)</Text>
+                <Text>{textarea.length}/14,400</Text>
+              </Flex>
+              <Box sx={styles.textArea}>
+                <Textarea
+                  rows={14}
+                  onDrop={handleDrop}
+                  onDragOver={handleDragOver}
+                  value={textarea?.value}
+                  onChange={handleTextArea}
+                />
+                <Text sx={styles.textAreaBottom}>Attach images by dragging & dropping, selecting or pasting them.</Text>
+              </Box>
+              <Text sx={{ marginTop: '10px', marginBottom: '10px', display: 'block' }}>Discussion (optional)</Text>
+              <Box sx={{ position: 'relative' }}>
+                <Box sx={{ position: 'absolute', bottom: '10px', left: '10px' }}>
+                  <Svg icon="website" width="15" />
+                </Box>
+                <Input
+                  placeholder="forum.balancer.fi/proposal..."
+                  onChange={handleChangeDiscussion}
+                  value={dicussion}
+                />
+              </Box>
+            </>
+          ) : (
+            <>
+              <Text sx={{ fontSize: '36px', fontWeight: '600px' }}>{title.length > 0 ? title : 'Untitled'}</Text>
+              <br />
+              <Text>{textarea?.value}</Text>
+            </>
```

Comment:
This should be a new component, we're supposed to render the markdown content and not textarea text in the second section

Reply:
Done but no upload functionality is there